ujcatapi/config.py

```diff
@@ -36,3 +36,8 @@ def _get_comma_separated_env_variable(name: str) -> List[str]:
 # ====== Feature flags ======
 ENABLE_FOO = _get_boolean_env_variable("ENABLE_FOO")
 ENABLE_BAR = _get_boolean_env_variable("ENABLE_BAR")
+
+ALLOWED_ORIGINS_REGEXPES = [
+    "http://localhost[^\\.]*$",
+    "capacitor://localhost$",
```

Comment:
```suggestion
```
`capacitor` is used for mobile so no need to add it to list of exceptions.

Reply:
Thanks for this information :)